modules/security-csrf/src/main/java/com/devonfw/module/security/csrf/common/api/exception/ApplicationBusinessException.java

```diff
@@ -0,0 +1,35 @@
+package com.devonfw.module.security.csrf.common.api.exception;
+
+import net.sf.mmm.util.nls.api.NlsMessage;
+
+/**
+ * Abstract base class for business exceptions of this application.
+ */
+public abstract class ApplicationBusinessException extends ApplicationException {
```

Comment:
We removed all this code from app templates. It is not needed for CSRF so please remove.